docs/current_docs/api/fs-filters.mdx

```diff
@@ -85,6 +85,14 @@ Here's an example of a Dagger Function that excludes everything in a given direc
 ```typescript file=./snippets/fs-filters/pre-call/typescript/index.ts
 ```
 
+</TabItem>
+<TabItem value="Java">
+
+Here's an example of a Dagger Function that excludes everything in a given directory except Java source code files:
+
+```typescript file=./snippets/fs-filters/pre-call/java/MyModule.java
```

Comment:
<img width="829" alt="Screenshot 2025-03-17 at 16 38 37" src="https://github.com/user-attachments/assets/c3300220-acf5-4962-80a9-2fcc3506ff46" />

Wrong snippet also why is it `typescript` and not java?

Reply:
🫣 oops

Fixed 👍